scalafix-core/src/main/scala/scalafix/rewrite/NoXml.scala

```diff
@@ -0,0 +1,42 @@
+package scalafix
+package rewrite
+
+import scala.meta._
+import patch.TreePatch.AddGlobalImport
+
+case object NoXml extends Rewrite {
+
+  override def rewrite(ctx: RewriteCtx): Patch = {
+    val patch = ctx.tree.tokens.collect {
+      case tok @ Token.Xml.Start() =>
+        ctx.addLeft(tok, "xml\"\"\"")
```

Comment:
Would it be possible to determine simple cases where single quotes would be enough? For small literals, I think many users would prefer single quotes 

```scala
// before
<br/>
// now
xml"""<br/>"""
// after
xml"<br/>"
```